### DISCUSSION TITLE
Update commerceguys/addressing to ^2.0.0

### DISCUSSION THREAD

**Discussion by bojanz** [craftcms/cms#14318] [Ideas] [answered]
I'm the developer behind commerceguys/addressing, I've been very happy to see Craft using it to power its address features.

Last year, after intense development, addressing got a [2.0.0 release](https://github.com/commerceguys/addressing/releases/tag/v2.0.0), bringing many improvements, and designed to be mostly backwards compatible (with the usage of the SubdivisionUpdater to convert existing data when an entity is loaded and saved).

So, I'd love to see Craft update to ^2.0.0 and help us drive the new branch forward via feedback and bug reports. Happy to answer any questions along the way.

**Comment by brandonkelly** [comment, marked as answer]
Hey @bojanz, thanks for pointing that out (and for all your hard work on the library)!

Today I pulled in commerceguys/addressing v2 for our upcoming 5.0 release (#14322).

**Reply by bojanz** [reply]
That's great news! Looking forward to v5.
